### DISCUSSION TITLE
How to generate notes in a relative folder

### DISCUSSION THREAD

**Discussion by NEmbgen** [mProjectsCode/obsidian-meta-bind-plugin#253] [Q&A] [answered]
Hi, I'm not sure if this is the right location. 

Is it possible to create a button with this plugin to create a note in the same directory or relative to it?

I have a note with the button below and want to create a note in a sibling folder. E.g:

- Folder A <- Parent folder
  - Note <- Note with Button
  - Session Notes <- Folder
     - New note goes here <- Note i want to create
    

I have it like this, but Meta Bind wants a path relative to the vault root.

```meta-bind-button
label: New Session Note
style: default
actions:
- type: templaterCreateNote
  templateFile: "_meta/templates/DnD_Session_Template.md"
  folderPath: "../Session Notes"
  openNote: true
```

How can I make the path dynamic?

**Comment by mProjectsCode** [comment, marked as answer]
As answered in discord today, no.

**Comment by francisHugo** [comment]
Is it possible to put the notes to the folderPath by using a template? just a thought, not tried.
